boilerplate/app/app.tsx.ejs

```diff
@@ -10,6 +10,9 @@ import { StorybookUIRoot } from "../storybook"
 import { RootStore, RootStoreProvider, setupRootStore } from "./models/root-store"
 import { BackButtonHandler, exitRoutes } from "./navigation"
 import { contains } from "ramda"
+import { enableScreens } from "react-native-screens"
+
+enableScreens();
```

Comment:
I'd like to add some more comments/documentation around this, but I'm confused myself. I have a question out to the core team about this.

Reply:
From Brent Vatne:

> enableScreens will put screens stacks in a view controller but createNativeStackNavigator actually uses UINavigationController

Maybe we should just say that.

```suggestion
// This puts screens in a native ViewController or Activity. If you want fully native
// stack navigation, use `createNativeStackNavigator` in place of `createStackNavigator`:
// https://github.com/kmagiera/react-native-screens#using-native-stack-navigator
enableScreens();
```